gravitee-apim-gateway/gravitee-apim-gateway-standalone/gravitee-apim-gateway-standalone-container/src/test/java/io/gravitee/gateway/standalone/flow/ExpressionLanguageConditionFlowTest.java

```diff
@@ -66,7 +65,8 @@ public void registerPolicy(ConfigurablePluginManager<PolicyPlugin> policyPluginM
     private boolean isJupiterMode() {
         final Configuration configuration = context.getBean(Configuration.class);
         return (
-            configuration.getProperty("api.jupiterMode.enabled", Boolean.class, false) && api.getExecutionMode() == ExecutionMode.JUPITER
+            configuration.getProperty("api.jupiterMode.enabled", Boolean.class, false) &&
```

Comment:
Should it be emulateV4Engine instead here ? (If so maybe the default value should be set to true ? :thinking: )
```suggestion
            configuration.getProperty("api.v2.emulateV4Engine.default", Boolean.class, false) &&
```

Reply:
I think the purpose of the old `api.jupiterMode.enabled` was just to enable jupiter and continue to fallback to the v3 engine or jupiter engine depending on the `api.executionMode`.
Sounds correct to remove this imho.